pymc3/tests/test_distributions.py

```diff
@@ -580,6 +581,14 @@ def test_beta(self):
                                  lambda value, alpha, beta: sp.beta.logpdf(value, alpha, beta))
         self.pymc3_matches_scipy(Beta, Unit, {'mu': Unit, 'sd': Rplus}, beta_mu_sd)
 
+    def test_kumaraswamy(self):
+        Rpositive = Domain([.01, .1, .9, .99, 1, 1.5, 2, 100, inf])
```

Comment:
Maybe move this to where other domains are defined?
I would consider using e-5 as first value, and also use this domain for Beta tests.

Reply:
… actually, the same holds for parameters of gamma, exponential, etc. so changing these tests to use this domain is perhaps better left for a subsequent merge.